iaso/api/profiles.py

```diff
@@ -744,5 +743,5 @@ def delete(self, request, pk=None):
 L'équipe {{site_name}}.</p>
     """
 
-    EMAIL_SUBJECT_FR = "Configurer un mot de passe pour votre nouveau compte sur {dns_domain}"
-    EMAIL_SUBJECT_EN = "Set up a password for your new account on {dns_domain}"
+    EMAIL_SUBJECT_FR = "Configurer un mot de passe pour votre nouveau compte sur {site_name}"
```

Comment:
I think we should use DNS_DOMAIN here too and not use current_site.domain at all

Reply:
OK, as you want!